src/router/db.ts

```diff
@@ -249,57 +249,56 @@ async function rpcCall(
   postTransaction?: PostTransactionCheck,
 ): Promise<any[] | undefined> {
   let rpcRes: any[] | undefined;
-  let errorOcurred = false;
-  let finallyErr;
+  let rpcErr;
```

Comment:
well, changes in this file should not be here since they have not been approved/merged. I need to fix a new bug that David spot deploying to staging so that will delay the changes.

Reply:
I checked that file out from `main` branch.